services/execsrv/store.go

```diff
@@ -48,7 +48,9 @@ func (p postgresStore) Create(projectID string, exec types.Exec) error {
 		return fmt.Errorf("failed to marshal spec to JSON: %w", err)
 	}
 
-	metadata, err := json.Marshal(&types.NodeMetadataV1{})
+	metadata, err := json.Marshal(&types.NodeMetadataV1{
```

Comment:
We should probably deprecate the `NodeMetadataV1` type. Probs should be `ExecMetadata`?